src/OpenTelemetry.Instrumentation.StackExchangeRedis/StackExchangeRedisCallsInstrumentationOptions.cs

```diff
@@ -44,5 +44,10 @@ public class StackExchangeRedisCallsInstrumentationOptions
         /// <para><see cref="IProfiledCommand"/>: the profiled redis command from which additional information can be extracted to enrich the activity.</para>
         /// </remarks>
         public Action<Activity, IProfiledCommand> Enrich { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the <see cref="StackExchangeRedisCallsInstrumentation"/> should enrich Activity with <see cref="ActivityEvent"/> entries about the Redis command processing/lifetime. Defaults to <see cref="bool">true</see>.
+        /// </summary>
+        public bool EnrichActivityWithEvents { get; set; } = true;
```

Comment:
I am not sure why the Events were added in the 1st place. There is no spec requirement about adding Events (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#redis), so it may be better to turn this feature off completely. 

If a user wants to have events, `Enrich` could be used.

Will wait to see if anyone has opinions on this.
(Note that the package is non-stable, so we are allowed to have breaking changes (i.e disable events)).

Reply:
@utpilla Can you add it to next SIG meeting agenda, and if no objections, we can remove the Events being populated in this instrumentation.

Reply:
Thanks for additional clarification. I see the need for events. But without a spec/sem. conventions, this is going to be done differently in diff. implementations.
Let me check if the spec has an issue tracking this..

Reply:
Events are a good fit i.m.o, but being able to turn them off would also be nice, since there is no way to remove events from Activities once added, so writing a processor doesn't work either.

Reply:
For now, we can add this flag. The package is anyway non-stable, so we would be okay to make breaking changes, if required (due to spec). Ideally, semantic conventions would list the name of the events, so at that, we'll need to influence the spec or change this instrumentation.
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.StackExchangeRedis